hdfs3/tests/test_hdfs3.py

```diff
@@ -54,6 +54,12 @@ def test_simple(hdfs):
         assert len(data) == len(out)
         assert out == data
 
+def test_no_conf():
+    os.environ.pop('HADOOP_CONF_DIR', '')
+    os.environ.pop('HADOOP_INSTALL', '')
+    conf = hdfs3.core.conf()
```

Comment:
Sorry, my fault: something needs to be added to the imports at the head of the file.